pom.xml

```diff
@@ -607,6 +607,14 @@
           <artifactId>maven-resources-plugin</artifactId>
           <version>3.0.1</version>
         </plugin>
+        <plugin>
+          <artifactId>maven-shade-plugin</artifactId>
+          <version>3.4.1</version>
+          <configuration>
+            <finalName>${jar.assembly.name.prefix}-spark${sparkbundle.version}_${scala.binary.version}-${project.version}-jar-with-dependencies</finalName>
```

Comment:
Could you add the operating system version here?  like ubuntu22.04

Reply:
do we want to make this change here? I saw assembly plugin original didn't contain this jar.assembly.name.prefix